x-pack/test/functional/apps/discover/value_suggestions_non_timebased.ts

```diff
@@ -17,12 +18,19 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await esArchiver.loadIfNeeded(
         'test/functional/fixtures/es_archiver/index_pattern_without_timefield'
       );
+      await kibanaServer.savedObjects.clean({ types: ['search', 'index-pattern'] });
+      await kibanaServer.importExport.load(
+        'test/functional/fixtures/kbn_archiver/index_pattern_without_timefield'
+      );
+      await kibanaServer.uiSettings.replace({ defaultIndex: 'without-timefield' });
```

Comment:
As you can see in `test/functional/fixtures/kbn_archiver/index_pattern_without_timefield.json` there are actually two index patterns in that archive `with-timefield` and `without-timefield`.  The previous test runs failed before the `with-timefield` index pattern ended up being the default one in Discover.  I added code here to set the default to the one the test needs (it's faster than having the test select it in the UI).